src/model/keychain.ts

```diff
@@ -230,6 +283,10 @@ export async function remove(key: string): Promise<void> {
   );
 
   try {
+    // remove from MMKV
+    keychainLocalStorage.delete(key);
```

Comment:
are you 100% sure deleting a non existing key doesn't throw? if that's the case then fine, otherwise the real keychain delete will never run